### PR TITLE
Upgrade to find-elm-dependencies 1.0.1

### PR DESCRIPTION
**Pull request by sentience**
This enables `node-elm-compiler#findElmDependencies` to correctly report all dependencies within a multi-source-directory project.

We need this for our Webpack build at Culture Amp, so I’m hoping this can be merged and a new version of node-elm-compiler released to NPM soon, so that we can then upgrade https://github.com/elm-community/elm-webpack-loader.

Thanks in advance!

### CODE REVIEW

**Comment by rtfeldman** [comment]
I'm ok with this if @eeue56 is!

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
   "homepage": "https://github.com/rtfeldman/node-elm-compiler",
   "dependencies": {
     "cross-spawn": "4.0.0",
-    "find-elm-dependencies": "0.18.1",
+    "find-elm-dependencies": "^1.0.0",
```
Comment (stoeffel) [inline package.json]: I would prefer an exact version `"1.0.0"`

**Comment by sentience** [comment]
All set, @stoeffel!

**Comment by eeue56** [comment]
This isn't good to go yet. There's some changes I'd like to happen to find-elm-dependencies first, I've commented on the PR there.

**Comment by sentience** [comment]
Updated to point at 1.0.1 in anticipation of NoRedInk/find-elm-dependencies#3 being merged.

**Comment by sentience** [comment]
Pushed a new version of this change to rebuild now that find-elm-dependencies 1.0.1 is published.

**Comment by sentience** [comment]
👋 @eeue56 Happy to merge and release this now?

**Comment by eeue56** [comment]
yep

**Comment by rtfeldman** [comment]
Published as `4.3.1` - thanks @sentience @eeue56 @stoeffel!

**Pull request merged**